include/swift/AST/ASTContext.h

```diff
@@ -865,6 +865,12 @@ class ASTContext {
   
   /// Whether our effective Swift version is in the Swift 3 family
   bool isSwiftVersion3() const { return LangOpts.isSwiftVersion3(); }
+  
+  /// Whether our effective Swift version is in the Swift 4 family
+  bool isSwiftVersion4() const { return LangOpts.isSwiftVersion4(); }
+  
+  /// Whether our effective Swift version is in the Swift 5 family
+  bool isSwiftVersion5() const { return LangOpts.isSwiftVersion5(); }
```

Comment:
Updated: I don't want `isSwiftVersion4` either :) We should be doing <, <=, >, >=, etc. checks against the major version.